batch7crud-roll9/src/main/java/com/lftechnology/batch7crud/services/StudentService.java

```diff
@@ -12,8 +14,16 @@
 public class StudentService {
   private StudentDAO stdDAO = new StudentDAO();
 
-  public Student save(Student student) throws DataException {
-    return stdDAO.insert(student);
+  public Student save(Student student) throws DataException, ValidationException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: !unknownSymbol!, !unknownSymbol! [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1160)